### DISCUSSION TITLE
717

### DISCUSSION THREAD

**Discussion by giscus** [jojoldu/blog-comments#899] [Comments]
# 717

2번의 연휴 동안 (5월 5,6,7 / 27,28,29) 도둑맞은 집중력을 읽었다. 외부 발표에서 산만함을 관리하기 위해 여러가지 시도를 하고 있다는 이야기를 했는데, 산만함과 집중력에 대해서 훨씬 더 정확하고 좋은 내용을 담고 있다는 이야기에 보게 되었다. 이 책이 특히나 좋았던 것은 명확하게 이것이 맞다 라고 이야기 하지 않는다. 반대 사례에 대해서도 이야기하면서 어디까지나 논문과 실험이고, 반대 의견도 있기 때문에 각자의 판단에 맡기는 것이 좋았다. 저자의 생각에 맞추기 위해 저자가 동의하는 부분만 발췌하는 다른 책들과 달라서 더 신뢰가 생기면서 읽었다. 아래는 이 책 전체에서 가장 기억에 남는 부분들만 정리했다. 좋은 내용이 워낙 많아서 가능하면 꼭 구매해서 전체를 읽어보시길 추천한다. 6장의 빅..

https://jojoldu.tistory.com/717

**Comment by hongji3354** [comment]
요새 집중력이 많이 고민이었는데 한 번 읽어봐야 겠네요. 감사합니다:)
본문 내용 중 나느시시즘이라는 단어가 있는데 나르시시즘이지 않나요?

**Reply by jojoldu** [reply]
앗 ㅋㅋㅋㅋ 오타 죄송합니다 ㅋㅋㅋ

**Comment by ironwhale1014** [comment]
안그래도 이 책 볼까 말까 고민했었는데 봐야 겠네요 좋은 글 감사합니다.

**Comment by AppleCEO** [comment]
가는 곳 마다 자신을 방송할 뿐 다른 정보는 수신하지 않는 사람들로 가득찬 느낌
좋지 않은 개발 문화를 가진 팀에 있었는데 위에 문구 공감가네요.
서로 자신의 방식대로 문서를 작성하고 다른 사람을 그 것을 따라가기도 힘든 상황이었네요...

**Reply by Voyager003** [reply]
범균님 유튜브에서도 이 책 추천하시던데 재밌겠네요 함봐야겠습니다!

**Comment by Voyager003** [comment]
범균님 유튜브에서도 이 책 추천하시던데 재밌겠네요 함봐야겠습니다!

**Comment by MarkSon-42** [comment]
영상에서 자주 뵙다가 이렇게 같은 책을 우연히 읽은점을 보고 허겁지겁 댓글 남깁니다:) 정리 정말 잘해주셨는데 특히 후기가 정말 제 마음속의 감상과 일치합니다..! 좋은 글 너무 감사합니다. (종종 올려주시는 도서 컨텐츠가 정말 꿀입니다..ㅎㅎ)

**Comment by ChoiJoongHyun** [comment]
꼭 읽어 봐야 겠네요 :)

**Comment by Crazy-HotDog** [comment]
향로님은 하루에 몇 시간씩 주무시나요? :)

**Comment by chrishoonS** [comment]
독서 같은 경우 정말 최근에 난독증이 참 화두가 되고 있는것 같은데 "난독의 시대"라는 책과 일맥상통하는 부분이 있는것 같네요. 좋은 책 정보 잘 보고 갑니다!

**Comment by giraffeb** [comment]
요새 이 책을 읽고 있었는데 같은 고민을 한다는 점이 반갑네요. 
공유 감사합니다~
